AppController/djinn.rb

```diff
@@ -5029,8 +5029,7 @@ def is_port_already_in_use(port_to_check)
   # Returns:
   def is_port_assigned(port_to_check)
     @pending_appservers.each { |appserver, _|
-      host, port = appserver.split(":")
-      next if @my_private_ip != host
```

Comment:
correct me if I'm wrong but, it seems like this check is necessary because the ip isn't checked at any other point in this specific part of the process and I think we could end up trying to use a port that is in use on the current machine but not on the host.